packages/react/src/card/card.tsx

```diff
@@ -185,13 +185,27 @@ export interface CardProps extends CardBaseProps {
    * as a link to the user. That would be perceived as information overload.
    */
   as?: "section" | "div" | "article" | "aside";
+  /**
+   * Allows for a horizontal variant for sizes above small.
+   *
+   * @default "default"
+   */
+  variant?: "default" | "horizontal";
```

Comment:
💡 If you need to get this published before we have landed the API / Shape of the new cards we can prefix this with `unstable_`, eg. `unstable_variant`. Then we don't need to do a major version update when iterating 👍